try_install_all.py

```diff
@@ -24,7 +24,8 @@
     ]
 SCRIPT_LIST = SCRIPT_LIST()
 TEST_ENGINES = {}
-IGNORE = ["AvianBodyMass", "FIA", "Bioclim", "PRISM", "vertnet"]
+IGNORE = ["AvianBodyMass", "FIA", "Bioclim", "PRISM", "vertnet","NPN", "mammsupertree", "eBirdOD"]
```

Comment:
The eBirdOD folks decided to tighten up on how people get their data and they moved it to another site. But we have vertnet which has ebird data.(for now)

In the case of NPN, we shall have to go through their sites, I guess a lot has changed on how to get data. They must have improved their service. I took a quick look and we need to discus what exactly we want to fetch from their repo. 


For this PR, I think ebird and NPN will take some time to get back to working state.

As we try to finish up the work on improving tests, It would be good if we can ignore those for now and solve the rest. The main goal is to have a working test on all scripts ASAP. 
And we shall set travis to run the test once major changes happen and periodically.